### ISSUE TITLE
Open new branch for processing 3

### ISSUE THREAD

**Issue by hanss314** [carykh/neuralNetworkLanguageDetection#12]
Only Cary can do this. If anyone can get him on discord it would be great.

**Comment by adamthedog** [comment]
He's normally on Discord around 3:00 for me (Midwest America), but anyways, Processing 3 replaced both P2D and P3D with OpenGL versions of it. That means it'll run faster (Java takes up a lot of memory and CPU usage), but rendering in 3D might be more complicated to get working well.

**Comment by adamthedog** [comment]
It'd be helpful if Cary gave us permissions to merge, make new branches, and such or if he'd actually watch what happens, though he does have to make videos and study for school.

**Comment by hanss314** [comment]
We're making two branchest because 2 and 3 have their own advantages and have different methods for scaling windows.

**Comment by hanss314** [comment]
I agree, some of the prs have been around for alost a week now and there's almost Ben merge conflict.

**Comment by abc123me** [comment]
I was able to get it working just by just changing one line:
`size((int)(monitorWidth*WINDOW_SCALE_SIZE),(int)(monitorHeight*WINDOW_SCALE_SIZE));`
to
`size(1360,` 768);`

I am not saying this is the best way, but it works
